locales/en/messages.json

```diff
@@ -1212,6 +1212,12 @@
     "configurationReceiverMode": {
         "message": "Receiver Mode"
     },
+    "configurationTelemetry": {
+        "message": "Telemetry"
+    },
+    "configurationTelemetryHelp": {
+        "message": "Telemetry output from receiver"
+    },
```

Comment:
Moving them here, I suppose maybe there is other place where we need to remove it? There are translations in this other place?
As I always recommend with new translations, please add a `description` with information to the translators. In this case I think is difficult to be wrong, but it always help.